backend/src/core/services/campaign.service.ts

```diff
@@ -24,7 +24,11 @@ const createCampaign = ({ name, type, userId }: {name: string; type: string; use
  * On file upload, save the transaction id and file name against the campaign so that we can download the file from s3 later
  * @param param0 
  */
-const updateCampaignS3Metadata = ({ key, campaignId, filename }: { key: string; campaignId: string; filename: string }): Promise<[number, Campaign[]]> => {
+const updateCampaignS3Metadata = (
```

Comment:
@chinying Is there a reason why the params was wrapped in an object previously? I could change it back.

Reply:
Just a personal preference, you don't have to go figure out in which order you have to enter your params when you have a lot of them